corehq/apps/userreports/indicators/specs.py

```diff
@@ -69,6 +69,14 @@ def getter(self):
         return TransformedGetter(getter, transform)
 
 
+class RepeatIterationIndicatorSpec(RawIndicatorSpec):
+    type = TypeProperty('repeat_iteration')
+
+    @property
+    def getter(self):
+        return lambda doc, context: context.iteration
```

Comment:
`"repeat_iteration"` indicators ignore the document and just return the iteration number from the context.